### DISCUSSION TITLE
uuidってどこで生成するんでしたっけ？

### DISCUSSION THREAD

**Discussion by masibw** [jphacks/F_2111#47] [Q&A]
今のサーバー実装見るとクライアント側からid送られてきそうなんですが，フロントでid作るんでしたっけ

**Comment by takurinton** [comment]
フロントエンドで S3 に画像をアップロードするときに画像を一意に識別するための id を生成するために uuid はフロントエンドで生成してるんだけど、データベースの id とは別かな？

**Reply by takurinton** [reply]
いや、でもオートインクリメントではないから、フロントエンドからもらった id をそのまま insert するイメージでいいのか

**Reply by masibw** [reply]
なるほどです！
idそのままDBに保存する実装になってますね

普通にサーバーサイドでid採番することが多い気はしますが別にフロントでも問題なさそうならおけです！
